### PR TITLE
Create  registration page with basic validation and form

### PR DESCRIPTION
**Pull request by nasreekar**
Also create placeholders for registration reducer file

### CODE REVIEW

**Comment by aosteraas** [comment]
Hey make sure you remove `package-lock.json` as we're using yarn in this project.

File: packages/client/src/locale/en.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import LocalizedStrings from 'react-localization';
+
+let strings = new LocalizedStrings({
```
Comment (aosteraas) [inline packages/client/src/locale/en.tsx:3]: I am pro localisation, how's the TypeScript support in this package?

File: packages/client/src/store/index.ts [hunk 2]
```diff
@@ -1,8 +1,10 @@
 import { configureStore, combineReducers } from '@reduxjs/toolkit';
 import { authReducer } from './auth';
+import { registrationReducer } from './registration'
 
 const rootReducer = combineReducers({
   authReducer,
+  registrationReducer,
```
Comment (aosteraas) [inline packages/client/src/store/index.ts:7]: Not a big deal but can you change these two lines to be

```
auth: authReducer,
registration: registrationReducer
```

File: packages/client/src/locale/en.tsx (hunk 1 above)
Comment (nasreekar) [reply packages/client/src/locale/en.tsx:3]: https://github.com/stefalda/react-localization/blob/master/README.md#typescript-support

File: packages/client/src/store/index.ts (hunk 2 above)
Comment (nasreekar) [reply packages/client/src/store/index.ts]: Fixed this 👍

File: packages/client/src/locale/en.tsx (hunk 1 above)
Comment (aosteraas) [reply packages/client/src/locale/en.tsx:3]: haha oh my god how did I not see that

**Pull request merged**